pandas/util/_test_decorators.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+def _safe_import(mod_name, min_version=None):
+    """
+    Parameters:
+    -----------
+    mod_name : str
+        Name of the module to be imported
+    min_version : str, default None
+        Minimum required version of the specified mod
+
+    Returns:
+    --------
+    object
+        The imported module if successful, or False
+    """
+    try:
+        mod = __import__(mod_name)
+    except ImportError:
+        return False
+
+    if not min_version:
+        return mod
+    else:
+        import sys
+        version = getattr(sys.modules[mod_name], '__version__')
+        if version:
+            from pkg_resources import parse_version as pv
```

Comment:
We typically use distutils but I think we require setup tools now so this is fine, right?

Reply:
Yes there is currently an entry for ``setuptools`` in ``requirements_dev.txt`` FWIW this was inspired by the current ``importorskip`` implementation in pytest that these skip functions were using previously